recipes/doxygen/all/conanfile.py

```diff
@@ -34,10 +34,15 @@ def configure(self):
         if minimum_compiler_version is not None:
             if tools.Version(self.settings.compiler.version) < minimum_compiler_version:
                 raise ConanInvalidConfiguration("Compiler version too old. At least {} is required.".format(minimum_compiler_version))
+        if (self.settings.compiler == "Visual Studio" and
+            tools.Version(self.settings.compiler.version.value) <= 14 and
+                tools.Version(self.version) == "1.8.18"):
+            raise ConanInvalidConfiguration("Doxygen version {} broken with VS {}.".format(self.version,
+                                                                                           self.settings.compiler.version))
         del self.settings.compiler.cppstd
```

Comment:
```suggestion
        del self.settings.compiler.libcxx
        del self.settings.compiler.cppstd
```

Reply:
@madebr I didn't change this in my PR, so this addition seems out of scope unless this is related to the other chaanges.

Reply:
Doxygen has c++ files, so libcxx is still useful for building only, because compiler is removed from package id.

Reply:
I'm seeing a lot of comments on this PR, and these all seem to be asking this PR to fix bugs that were already present.  This issue wasn't addressed or intended to be addressed here, the same actually applies to the above version combination of visual studio and Doxygen that breaks.  IMO those could be put in separate issues and resolved separately by people who understand these problems.